README.md

```diff
@@ -63,64 +89,103 @@ Untracked files:
 nothing added to commit but untracked files present (use "git add" to track)
 ```
 
-Git is telling you that this is a brand new repository and right now Git isn't keeping track of any of the files in your directory.
+Git is telling we that this is a brand new repository and right now Git
+isn't keeping track of any of the files in our directory.
 
-Whenever you want to check the status of your Git repository –– which you'll probably do quite often –– type `git status`. We'll discuss the various states a repository can be in and how to change states shortly.
+Whenever we want to check the status of our Git repository –– which
+we'll probably do quite often –– type `git status`. 
 
-## Keeping track of files with `git add`
+## Recognize How to Keep Track of Files with `git add`
 
-You have to explicitly tell Git about all the files you want it to keep track of– the files you want Git to consider as part of your project. We do this by adding the files to our Git repository with `git add <filename or path>`. To add our new `README.md` to the repository and check the status, we would type:
+Currently the files in our repository are not being tracked by Git just yet.
+We have to tell Git about all the files we want it to keep track
+of and consider as part of our project. We can do this by adding the files
+to our Git repository with `git add <filename or path>`. To add our new
+`README.md` to the repository and check the status, we would type:
 
 ```
-next-big-thing $ git add README.md
-next-big-thing $ git status
+my-git-project $ git add README.md
+my-git-project $ git status
 On branch master
 
-Initial commit
+No commits yet
 
 Changes to be committed:
   (use "git rm --cached <file>..." to unstage)
 
 	new file:   README.md
 ```
 
-You can now see that Git is ready to keep track of `README.md` and recognizes that the file is new to this repository. However, while we've informed Git that there is a new file, we still haven't informed Git that this new file is considered a change to our repository. We create changes in our repository by making "commits."
+We can now see that Git is ready to keep track of `README.md` and recognizes
+that the file is new to this repository. However, while we've informed Git that
+there is a new file, we still haven't informed Git that this new file is
+considered a change to our repository. We create changes in our repository
```

Comment:
this new file's current state should be "snapshotted." 

we create changes, or "snapshot,"...